resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -85,6 +85,12 @@ private[spark] object Config extends Logging {
       .stringConf
       .createOptional
 
+  val KUBERNETES_EXECUTOR_CORES =
+    ConfigBuilder("spark.kubernetes.executor.cores")
+      .doc("Specify the cpu request for each executor pod")
+      .stringConf
+      .createOptional
```

Comment:
Can we use `fallbackConf` here?

Reply:
Unfortunately, there's not a `ConfigEntry` for `spark.executor.cores` in core.